testing/tests/DevExpress.ui.widgets/overlay.tests.js

```diff
@@ -855,6 +855,30 @@ testModule('visibility', moduleConfig, () => {
         }
     });
 
+    if(!QUnit.isInShadowDomMode()) {
+        test('overlay should not close after click on item in shadow dom (T1146455)', function(assert) {
```

Comment:
Let's specify that we're talking only about element inside an overlays's content. Overlay should still close after click on element in shadow dom if this element is not inside a content.
```suggestion
        test('overlay should not close after click on content element in shadow dom (T1146455)', function(assert) {
```